docs/edgeql/transactions.rst

```diff
@@ -42,12 +42,25 @@ under the hood.
 TypeScript/JS
 ^^^^^^^^^^^^^
 
+Using an EdgeQL query string:
+
 .. code-block:: typescript
 
   client.transaction(async tx => {
     await tx.execute(`insert Fish { name := 'Wanda' };`);
   });
 
+Using the querybuilder:
+
+.. code-block:: typescript
+
+  const query = e.insert(e.Fish, {
+    name: 'Wanda'
+  });
+  client.transaction(async tx => {
+    await query.run(tx);
+  });
+
```

Comment:
I think you should also take a look at edgedb-js docs specifically and see if we can improve them too